src/main/java/ru/mts/Main.java

```diff
@@ -1,20 +1,18 @@
 package ru.mts;
-import shop.Product;
 
-public class Main {
+import ru.mts.animalsCreators.CreateAnimalServiceLmpl;
 
-    public static void printSumOfPurchase(Product product)
-    {
-        System.out.println("Сумма покупки без скидки: " + product.sum * product.quantity + " Со скидкой: " + String.format("%.2f", product.getDiscountSum()));
-    }
+public class Main {
     public static void main(String[] args) {
 
-        Product Bread = new Product(10,25,0.75),
-                Water = new Product(5,30,42.575),
-                Butter = new Product(25,100,59.1);
+        int N = 4;
```

Comment:
в джаве названия переменных, равно как и методов - нижний регистр

```suggestion
        int n = 4;
```